lineman/app/models/discussion_model.coffee

```diff
@@ -109,7 +109,7 @@ angular.module('loomioApp').factory 'DiscussionModel', (BaseModel, AppConfig) ->
       @remote.patchMember @keyOrId(), 'set_volume', { volume: volume }
 
     toggleStar: ->
-      @remote.patchMember @keyOrId(), if @starred then 'unstar' else 'star'
```

Comment:
It'd also be interesting to see what 'debounce' did here.. in a perfect world it would provide a little buffer time so that the web request can fire off before we update the model locally.

```
input{ng-model: 'thread.starred', ng-model-options: '{debounce: 100}'}
```